src/main/scala/gitbucket/core/controller/RepositoryViewerController.scala

```diff
@@ -974,7 +974,9 @@ trait RepositoryViewerControllerBase extends ControllerBase {
             createIssueComment(repository.owner, repository.name, commitInfo)
 
             // close issue by commit message
-            closeIssuesFromMessage(message, loginAccount.userName, repository.owner, repository.name)
+            if (branch == repository.repository.defaultBranch) {
```

Comment:
I'm not sure what exactly the problem is. It should at least allow to close issue on non-default branch as well.  When two non-default branches are to be merge together other than default branch. Even from a fork too.

My question is, it does not impact with non-default branch merge to another non-default branch?